packages/nitro-protocol/docs/force-move-safety.md

```diff
@@ -0,0 +1,399 @@
+| Max       | Width                | Table          | Width        |
+| --------- | -------------------- | -------------- | ------------ |
+| Open(n,S) | forceMove(m,t,s,p,S) | C(m,t+exp,s,p) | valid(s->s') |
+
+This is the maxiumum width of a nicely formatted table in a medium post.
+
+# Intro
+
+ForceMove is a state-channel protocol, designed to be a simple as possible.
+A set of participants exchanging states in private, which determine some outcome of the state channel.
+An adjudicator, in the form of a smart contract, enforces the outcome on a blockchain, which would typically involve some financial transaction.
+As such, the participants wish for the system to protect against malicious actors that try to prevent the most fair outcome from being recorded on-chain.
+
+At Magmo, we have spent some time specifying the ForceMove protocol in [ TLA+ ](https://en.wikipedia.org/wiki/TLA%2B), a formal specification language based on set theory.
+Engineers at Amazon Web Services [have used](https://lamport.azurewebsites.net/tla/formal-methods-amazon.pdf) TLA+ since 2011 when designing some of their critical systems\*.
+
+# ForceMove overview
+
+<!-- TODO: I'm not sure what to say in the general overview -->
+<!-- Currently:
+1. A ForceMove state channel is a state machine that determines the distribution of assets
+2. Valid transitions, turn numbers, ordered turns
+3. Adjudicator's role
+4. Supported states
+-->
+
+In a ForceMove state channel, participants, privately update the state of some ledger.
```

Comment:
IMO 'ledger' seems a bit outdated these days and I don't see it used an awful lot in the ethereum community.

```suggestion
In a ForceMove state channel, a small number of participants update a private state that is shared between them.
```